xtra-libp2p-ping/src/ping.rs

```diff
@@ -128,11 +127,7 @@ impl Actor {
                 tracing::warn!(%peer, "Outbound ping protocol failed: {e:#}")
             };
 
-            spawn_fallible(
-                &this,
-                ping_fut.instrument(tracing::debug_span!("Ping peer", %peer)),
```

Comment:
I think this will probably still result in spans being produced:
- for the second to the endpoint
- for the send_async_next

They just wouldn't be grouped together anymore. 

If the goal is to make them not be children of the current span, you could replace the `debug_span!` with `debug_span!(parent: Span::none(), "Ping peer", %peer)`, I think

Reply:
if it doesn't remove the span, I'd consider removing this part of the changeset (the goal here was to remove some traces).

Reply:
it did remove the additional `peer_id` though.